plugins/SedRegex/config.py

```diff
@@ -57,6 +57,17 @@ def configure(advanced):
 conf.registerChannelValue(SedRegex, 'ignoreRegex',
     registry.Boolean(True, _("""Should Perl/sed regex replacing
                      ignore messages which look like valid regex?""")))
+conf.registerChannelValue(SedRegex, 'sedString',
```

Comment:
I would name this variable `formatString` and the other one `formatString.other` so that their meaning is clearer just from their names. (To register a variable under another one you could do something like `conf.registerChannelValue(SedRegex.formatString, ...)`

LGTM otherwise!

Reply:
Just `format` instead of `formatString`